pyrgg/params.py

```diff
@@ -1,22 +1,45 @@
 # -*- coding: utf-8 -*-
 """Pyrgg params."""
+from textwrap import dedent, fill
```

Comment:
If it's necessary you should add this library to `requirements.txt` too.

Reply:
`textwrap` is a built-in library in Python

Reply:
OK so no need to merge.